libs/storage/Tsavorite/cs/src/core/Index/Interfaces/TsavoriteEqualityComparer.cs

```diff
@@ -39,23 +38,20 @@ public static ITsavoriteEqualityComparer<T> Get<T>()
     public sealed class StringTsavoriteEqualityComparer : ITsavoriteEqualityComparer<string>
     {
         /// <summary>
-        /// Equals
+        /// The default instance.
         /// </summary>
-        /// <param name="key1"></param>
-        /// <param name="key2"></param>
-        /// <returns></returns>
+        /// <remarks>Used to avoid allocating new comparers.</remarks>
+        public static readonly StringTsavoriteEqualityComparer Instance = new();
+
+        /// <inheritdoc />
         public bool Equals(ref string key1, ref string key2)
         {
             // Use locals in case the record space is cleared.
             string k1 = key1, k2 = key2;
             return (k1 is null || k2 is null) ? false : k1 == k2;
         }
```

Comment:
Note for future: It may be worth it to investigate whether we should offer Ordinal string comparer as option consumers that know their keys are strictly culture-agnostic. This currently has little bit overhead for all the culture stuff.

Reply:
Tsavorite string types aren't used by Garnet, thankfully.